### PR TITLE
dont run pyspy on darwin

### PR DESCRIPTION
**Pull request by tarekziade**
tweak to not use `--pyspy` on darwin

### CODE REVIEW

File: connectors/tests/ftest.sh [hunk 1]
```diff
@@ -7,26 +7,39 @@ PERF8=$2
...
 cd $ROOT_DIR/connectors/sources/tests/fixtures/$NAME
 make run-stack
+
 # XXX make run-stack should be blocking until everythign is up and running by checking hbs
 sleep 30
 
 $ROOT_DIR/bin/fake-kibana --index-name search-$NAME --service-type $NAME --debug
 
-make load-data
+#make load-data
```
Comment (seanstory) [inline connectors/tests/ftest.sh:27]: Should this still be commented out?

File: connectors/tests/ftest.sh [hunk 2]
```diff
@@ -7,26 +7,39 @@ PERF8=$2
...
 if [[ $PERF8 == "yes" ]]
 then
-    $ROOT_DIR/bin/perf8 --memray --pyspy --psutil -c $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
-
+    if [[ $PLATFORM == "darwin" ]]
```
Comment (seanstory) [inline connectors/tests/ftest.sh:30]: The description says
> don't run pyspy on darwin

but this _only_ runs pyspy on darwin. Should this be a `-ne`?

Comment (tarekziade) [reply connectors/tests/ftest.sh:30]: ahah good catch thanks :)

File: connectors/tests/ftest.sh (hunk 1 above)
Comment (tarekziade) [reply connectors/tests/ftest.sh:27]: debug left over, will fix

**Comment by artem-shelkovnikov** [comment]
What's up with Darwin? It's not available on it?

**Comment by tarekziade** [comment]
> What's up with Darwin? It's not available on it?

it requires root privileges, and even when running as a root, I had errors and problems.

**Review by artem-shelkovnikov — APPROVED** [review]

**Pull request merged**